project_issue_task/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 {
     'name': 'Project Issue related Tasks',
     'summary': 'Use Tasks to support Issue resolution reports',
-    'version': '9.0.1.0.1',
+    'version': '9.0.1.0.2',
```

Comment:
Use `9.0.1.1.0`, because a server restart is not enough - a module upgrade is needed.